packages/synapse-react-client/test/lib/containers/oauth/CreateOAuthClient.integration.test.tsx

```diff
@@ -53,7 +57,7 @@ function setUp(props: CreateOAuthModalProps = defaultProps) {
   const component = renderComponent(props)
   const inputs = {
     name: screen.getByRole('textbox', { name: 'Client Name' }),
-    homePage: screen.getByRole('textbox', { name: 'Client Homepage' }),
+    homePage: screen.getByLabelText('Client Homepage'),
```

Comment:
Use `getByLabelText` to find this field, as discussed [here](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/212#discussion_r1190191428)